bin/start_fe.sh

```diff
@@ -163,6 +163,10 @@ fi
 for f in $STARROCKS_HOME/lib/*.jar; do
   CLASSPATH=$f:${CLASSPATH};
 done
+# add custom_libs to CLASSPATH
+for f in $STARROCKS_HOME/custom_lib/*.jar; do
+    CLASSPATH=$f:${CLASSPATH}
+done
```

Comment:
ditto

Reply:
> ditto

start_fe.sh and start_backend.sh load custom_lib commit removed